examples/upload_file.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+// require_once('./init.php');
+require_once('vendor/autoload.php');
+
+// set these environment variables
+$jwplatform_api_key = 'APIKEY';
+$jwplatform_api_secret = 'APISECRET';
+
+$jwplatform_api = new Jwplayer\JwplatformAPI($jwplatform_api_key, $jwplatform_api_secret);
+
+$params = array();
+$params['title'] = 'PHP API Test Upload';
+$params['description'] = 'Video description here';
+
+// Create the example video
+$create_response = json_encode($jwplatform_api->call('/videos/create', $params));
+
+$decoded = json_decode(trim($create_response), TRUE);
+$upload_link = $decoded['link'];
+
+$url = $upload_link['protocol'] . '://' . $upload_link['address'] . $upload_link['path'] .
```

Comment:
I would replace `$upload_link['protocol']` with `https` since the current default might be `http` which we want to discourage.

Reply:
hey Kamil, I just pushed up another commit and I changed the name of the script to make it more distinct and I hard coded in the protocol as you had suggested.